sensor_msgs/msg/Attitude.msg

```diff
@@ -0,0 +1,14 @@
+# This message represents 3D attitude
+
+int8 ENU=1
+int8 NED=2
```

Comment:
Why not enforce one of these?

Now all subscribers should handle this. I would like it better when there is only one definition within the ROS ecosystem.

Reply:
Stating in the message header-comment that ENU convention should be used.